tests/bluechi_test/test.py

```diff
@@ -113,6 +134,25 @@ def teardown(self, ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNod
         for _, node in nodes.items():
             node.cleanup()
 
+    def check_valgrind_logs(self) -> None:
+        logger.debug("Checking valgrind logs...")
+        errors_found = False
```

Comment:
Should we check first if self.tmt_test_data_dir exists? 

if ! os.path.exists(self.tmt_test_data_dir):
    return None

Reply:
Nope, the dir is already there since we already copied logs there. If it doesn't exist - something went wrong, and we will get errors which is a good thing - this is how we will know something is broken